hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieTableServiceClient.java

```diff
@@ -541,15 +582,15 @@ private void completeClustering(HoodieReplaceCommitMetadata metadata,
       // Only in some cases conflict resolution needs to be performed.
       // So, check if preCommit method that does conflict resolution needs to be triggered.
       if (isPreCommitRequired()) {
-        preCommit(metadata);
+        preCommit(replaceCommitMetadata);
       }
       // Update table's metadata (table)
-      writeTableMetadata(table, clusteringInstant.requestedTime(), metadata);
+      writeTableMetadata(table, clusteringInstant.requestedTime(), replaceCommitMetadata);
 
       LOG.info("Committing Clustering {} for table {}", clusteringCommitTime, table.getConfig().getBasePath());
-      LOG.debug("Clustering {} finished with result {}", clusteringCommitTime, metadata);
+      LOG.debug("Clustering {} finished with result {}", clusteringCommitTime, replaceCommitMetadata);
```

Comment:
Should move the debug log under `ClusteringUtils.transitionClusteringOrReplaceInflightToComplete`.